src/main/fc/fc_rc.c

```diff
@@ -259,9 +265,14 @@ FAST_CODE uint8_t processRcInterpolation(void)
 }
 
 #ifdef USE_RC_SMOOTHING_FILTER
-uint8_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline)
+uint16_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline, bool pt1)
```

Comment:
Let's get into habit to not use specific-size types for passing variables unless necessary.
You don't gain anything by going `uint16_t` here, just use `int`.